lib/Swagger/src/SiteAttribute/Api/SiteAttributeInternalApi.php

```diff
@@ -0,0 +1,217 @@
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * NOTE: This class is auto generated by the swagger code generator program.
+ * https://github.com/swagger-api/swagger-codegen
+ * Do not edit the class manually.
+ */
+
+namespace Swagger\Client\SiteAttribute\Api;
+
+use \Swagger\Client\Configuration;
+use \Swagger\Client\ApiClient;
+use \Swagger\Client\ApiException;
+use \Swagger\Client\ObjectSerializer;
+
+/**
+ * SiteAttributeInternalApi Class Doc Comment
+ *
+ * @category Class
+ * @package  Swagger\Client
+ * @author   http://github.com/swagger-api/swagger-codegen
+ * @license  http://www.apache.org/licenses/LICENSE-2.0 Apache License v2
+ * @link     https://github.com/swagger-api/swagger-codegen
+ */
+class SiteAttributeInternalApi
+{
+
+    /**
+     * API Client
+     *
+     * @var \Swagger\Client\ApiClient instance of the ApiClient
+     */
+    protected $apiClient;
+
+    /**
+     * Constructor
+     *
+     * @param \Swagger\Client\ApiClient|null $apiClient The api client to use
+     */
+    public function __construct(\Swagger\Client\ApiClient $apiClient = null)
+    {
+        if ($apiClient == null) {
+            $apiClient = new ApiClient();
+            $apiClient->getConfig()->setHost('https://localhost/site-attribute');
+        }
+
+        $this->apiClient = $apiClient;
+    }
+
+    /**
+     * Get API client
+     *
+     * @return \Swagger\Client\ApiClient get the API client
+     */
+    public function getApiClient()
+    {
+        return $this->apiClient;
+    }
+
+    /**
+     * Set the API client
+     *
+     * @param \Swagger\Client\ApiClient $apiClient set the API client
+     *
+     * @return SiteAttributeInternalApi
+     */
+    public function setApiClient(\Swagger\Client\ApiClient $apiClient)
+    {
+        $this->apiClient = $apiClient;
+        return $this;
+    }
+
+    /**
+     * Operation internallySaveAttribute
+     *
+     * Saves an attribute for a specified site
+     *
+     * @param int $site_id The id of the site (required)
+     * @param string $attr_name The name of the attribute to be saved (required)
+     * @param string $x_wikia_schwartz  (optional)
+     * @param \Swagger\Client\SiteAttribute\Models\SiteAttributeValue $body  (optional)
+     * @return \Swagger\Client\SiteAttribute\Models\SiteAttributeHalResponse
+     * @throws \Swagger\Client\ApiException on non-2xx response
+     */
+    public function internallySaveAttribute($site_id, $attr_name, $x_wikia_schwartz = null, $body = null)
+    {
+        list($response) = $this->internallySaveAttributeWithHttpInfo($site_id, $attr_name, $x_wikia_schwartz, $body);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Immediately return this expression instead of assigning it to the temporary variable "$response". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1488)